docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -254,7 +254,12 @@ Specify where to output log information. Valid options are:
 * `DEFAULT_BOTH`
 * `FILE`
 
-Defaults to `DEFAULT_BOTH`.
+Defaults to `DEFAULT_BOTH`. When using `BOTH` or `DEFAULT_BOTH`, system updates such as blockchain events 
+are displayed on the console, and errors and other information are logged to a file. The log file
+location can be specified with the [`--log-file`](#log-file) command-line option.
+
+For production systems we recommend using the `CONSOLE` or `FILE` options. When using `CONSOLE` you can
+configure `systemd` to handle the output.
```

Comment:
I'd probably remove the systemd comment.  How you manage the output depends on what system you're using. Kind of want to explain why we recommend console or file though - primarily it's about simplicity as there's now only one place to look for logs.  CONSOLE is particularly well suited to have other systems (such as systemd or docker) manage the logging output.

```suggestion
For production systems we recommend using the `CONSOLE` or `FILE` options to ensure all log information is available in one place.
```